.github/workflows/cd-syft.yml

```diff
@@ -131,6 +131,35 @@ jobs:
             echo "github_release_version=$(python packages/grid/VERSION)" >> $GITHUB_OUTPUT
           fi
 
+      # Checkout Infra repo (nested)
+      - name: Checkout Infra Repo
+        uses: actions/checkout@v3
+        with:
+          repository: OpenMined/infrastructure
+          token: ${{ secrets.InfraRepoToken }}
+          path: infrastructure
+
+      # This step will copy the generated K8s manifest files to the correct directory in Infra repo
+      - name: Copy files to Repo B
+        run: |
+          if $is_pre_release; then
+            rm -rf infrastructure/gitops/environments/pre-release/.
+            cp -R packages/grid/helm/syft/. packages/grid/helm/manifest.yaml infrastructure/gitops/environments/pre_release/
+          else
+            rm -rf infrastructure/gitops/environments/stable/.
+            cp -R packages/grid/helm/syft/. packages/grid/helm/manifest.yaml infrastructure/gitops/environments/stable/
+          fi
+
+      # This step will commit and push changes Infra repo
+      - name: Push to Infra Repo
+        run: |
+          cd infrastructure
+          git config user.name "bitsofsteve"
+          git config user.email "cyberlexix@gmail.com"
+          git add .
+          git commit -m "Update K8s manifests from Syft repo"
+          git push
+
```

Comment:
This can be:
```
      - name: Commit & push changes to Infra Repo
        uses: EndBug/add-and-commit@v7
        with:
          author_name: Madhava Jay
          author_email: madhava@openmined.org
          message: "[env] Update helm charts"
          cwd: infrastructure
          pull: NO-PULL
```

Reply:
Additionally, this should be done using an automation account - with the ability to pull&push across the Syft.  cc: @rasswanth-s

Reply:
Done 👍